src/Main.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+        Integer[]numeric = {1,2,3,4,5,6,7,8,9,0};
+        String[]alphabetic = {"een","twee","drie","vier", "vijf","zes","zeven","acht","negen","nul"};
+        Translator translator = new Translator(alphabetic,numeric);
+        String answer = translator.translate(3);
+        System.out.println(answer);
+
+        boolean play = true;
+        String ongeldig = "ongeldige invoer";
+        Scanner scanner = new Scanner(System.in);
+        while (play) {
+            System.out.println("Type 'x' om te stoppen \nType 'v' om te vertalen");
+            String input = scanner.nextLine();
+                    if (input.equals("x")) {
```

Comment:
Misschien handig (als je het nodig acht) om ook eens `.equalsIgnoreCase( ) te gebruiken. voor het geval dat er mensen een hoofdletter geven als input. Dan dek je jezelf daartegen in.

Reply:
Thnx voor de tip! Goede!